src/index.js

```diff
@@ -0,0 +1,127 @@
+import './styles.css';
+import 'bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import $ from 'jquery';
+import moment from 'moment';
+
+const dayGIF = 'https://mondrian.mashable.com/wp-content%252Fuploads%252F2013%252F04%252Fpalm-trees.gif%252Ffull-fit-in__1200x2000.gif?signature=u6Pkj2nxfqX8m9jTaqwq1Xl6iqM=&source=http%3A%2F%2Fmashable.com';
+const nightGIF = 'https://i.gifer.com/VTNI.gif';
+
+async function getCity() {
+  const cityInput = $('.city-name');
+  const cityName = cityInput.val();
+  cityInput.val('');
+
+  const celOrFahrNode = document.querySelector('.cel-or-fahr');
+  const celOrFahr = celOrFahrNode.textContent;
+  let API = '';
+
+  if (celOrFahr === 'celsius') {
+    API = `https://api.openweathermap.org/data/2.5/weather?q=${cityName}&units=metric&appid=934ea5f8bc3f198fabf59b607a2fcc71`;
+  } else {
+    API = `https://api.openweathermap.org/data/2.5/weather?q=${cityName}&units=imperial&appid=934ea5f8bc3f198fabf59b607a2fcc71`;
+  }
+
+  const response = await fetch(API);
+  const city = await response.json();
+
+  return city;
+}
+
+function displayDetails(city) {
+  const cityName = $('.content-city-name');
+  const cityTemperature = $('.content-city-temperature');
+  const cityTime = $('.content-city-time');
+  const celOrFahr = $('.cel-or-fahr').text();
+  const cityTemperatureType = $('.content-city-temperature-type');
+
+  const { dt } = city;
+  const cityTimezone = city.timezone;
+
+  const currentTime = moment.unix(dt).utc().add(cityTimezone, 's');
+  const currentTimeReadable = currentTime.format('LTS');
+
+  cityName.empty();
+  cityTemperature.empty();
+  cityTime.empty();
+  cityTemperatureType.empty();
+
+  cityName.append(city.name, ', ', city.sys.country);
+  cityTemperature.append(Math.round(city.main.temp));
+
+  if (celOrFahr === 'celsius') {
+    cityTemperatureType.append('C');
+  } else {
+    cityTemperatureType.append('F');
+  }
+
+  cityTime.append(currentTimeReadable);
+
+  return currentTime._i;
```

Comment:
- [ ]  As this is what is causing the eslint error then consider disabling the eslint no-underscore-dangle rule for this particular line. Only do this if there is no other way to get the project to work without this underscore.